src/components/PingChart.vue

```diff
@@ -182,7 +182,7 @@ export default {
             // eslint-disable-next-line eqeqeq
             if (newPeriod == "0") {
                 this.heartbeatList = null;
-                this.$root.storage().removeItem(`chart-period-${this.monitorId}`);
+                this.$root.storage().removeItem("chart-period");
```

Comment:
I don't think this quite makes sense.
Having two default states is defintively a bug.

```suggestion
                this.$root.storage()["chart-period"] = newPeriod;
```

Reply:
At least I don't see a reason why we would call `removeItem` in this case